packages/orchestration/README.md

```diff
@@ -68,6 +69,23 @@ const response = await orchestrationClient.chatCompletion({
 const responseContent = response.getContent();
 ```
 
+#### Passing a custom request configuration
+
+To pass custom request configuration, for example additional headers to be passed to the client, use the following snippet:
```

Comment:
```suggestion
To customize the request configuration, for example if you want to pass additional headers to the client, use the second parameter in the `chatCompletion()` method:
```

Reply:
[pp] Maybe also add a sentence that this configuration takes precedence over automatically created configuration for your request. 
Also, once there is more information on deployment resolution, make sure to mention that the request config is not used for deployment resolution.